libraries/chain/database.cpp

```diff
@@ -2259,6 +2260,44 @@ namespace golos { namespace chain {
             }
         }
 
+        uint64_t database::pay_delegators(const account_object& delegatee, const comment_vote_object& cvo, uint64_t claim) {
+            uint64_t delegators_reward = 0;
+            const auto& vdo_idx = get_index<vesting_delegation_index>().indices().get<by_received>();
+            const auto& vdeo_idx = get_index<vesting_delegation_expiration_index>().indices().get<by_received>();
+            for (auto& dvir : cvo.delegator_vote_interest_rates) {
+                auto delegator_claim = claim * dvir.interest_rate / STEEMIT_100_PERCENT;
+
+                if (delegator_claim == 0) {
+                    continue;
+                }
+
+                delegators_reward += delegator_claim;
+
+                const auto& delegator = get_account(dvir.account);
+                asset delegator_vesting = create_vesting(delegator, asset(delegator_claim, STEEM_SYMBOL));
```

Comment:
@afalaleev What is "direct sending to delegator"? We already sending to delegator's v_s, d_v_s and r_v_s, no matter exists VDO and VDOE or not.
Virtual operation is a good idea.

Reply:
Vesting can't be send from one account to other, it can only be delegated.
That is why you do the following steps:
- Create vesting on delegator balance: `create_vesting(delegator, asset(delegator_claim, STEEM_SYMBOL));`
- Save information about delegated vesting: `a.delegated_vesting_shares += delegator_vesting;`

But in this case, delegator should request to return of vesting.
That is why, this strategy is `to_delegated_vesting`.
Main advantage of the strategy that delegatee can use this vesting on next vote.

But some delegators want to receive percents from delegated vesting without requesting it from delegatee.
This strategy will be named `to_delegator`. To implement it you shouldn't save information about delegated vesting, you should only create vesting on balance of delegator.